.travis.yml

```diff
@@ -1,11 +1,10 @@
 language: python
 python:
-  - "3.6"
-  - "3.6-dev"  # 3.6 development branch
   - "3.7"
   - "3.7-dev" # 3.7 development branch
 install:
-  - pip install -r requirements.txt --no-cache
+  - pip install poetry
```

Comment:
From the poetry docs, the recommended method to install poetry is not to use pip rather their own script